common/app/views/fragments/header.scala.html

```diff
@@ -26,6 +26,40 @@
                             <span class="control__info js-control-info">register</span>
                         </a>
                         <span class="u-h brand-bar__item__badge">free</span>
+                        <div class="u-h js-popup-polly-toynbee-quote popup popup--text popup--text--register">
+                            <img src="/assets/images/polly-toynbee.png"
```

Comment:
Why dod you need a custom image that we host here, rather than:
http://i.guim.co.uk/static/w-115/h--/q-95/sys-images/Guardian/Pix/contributor/2014/6/30/1404146756739/Polly-Toynbee-L.png